packages/components/src/text/__tests__/Text.web.test.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+import * as React from 'react';
+import { Platform } from 'react-native';
+import { render } from 'react-native-testing-library';
+import { Text as UniversalComponentsText } from '@kiwicom/universal-components';
+
+import Text from '../Text';
+
+const originalPlatform = Platform.OS;
+
+beforeAll(() => {
+  Platform.OS = 'web';
```

Comment:
Maybe we should just use `jest.mock` as in the other tests?

Reply:
There was some issue when running Jest; I think react-native-web messes up with `jest.mock('Platform', () => ...)`. I can have another look.

Reply:
No, I am ok with this solution as well